java/generate_screenshot/README.md

```diff
@@ -0,0 +1,61 @@
+# 💻 Get Short URL
+
+A Java Cloud Function to get price of crypto.
```

Comment:
This line should be rectified to mention that this function is used to generate srceenshots, also mention the file types supported